### ISSUE TITLE
📊Tracking: AWS DMS

### ISSUE THREAD

**Issue by SomayaB** [aws/aws-cdk#6906]
Add your +1 👍  to help us prioritize high-level constructs for this service
--- 

### Overview:

AWS Database Migration Service (AWS DMS) is a cloud service that makes it easy to migrate relational databases, data warehouses, NoSQL databases, and other types of data stores. You can use AWS DMS to migrate your data into the AWS Cloud, between on-premises instances (through an AWS Cloud setup), or between combinations of cloud and on-premises setups.

[AWS Docs](https://docs.aws.amazon.com/dms/latest/userguide/Welcome.html)  

### Maturity: CloudFormation Resources Only

See the [AWS Construct Library Module Lifecycle doc](https://github.com/aws/aws-cdk-rfcs/blob/master/text/0107-construct-library-module-lifecycle.md) for more information about maturity levels.

### Implementation:

See the [CDK API Reference](https://docs.aws.amazon.com/cdk/api/latest/docs/aws-dms-readme.html) for more implementation details.

### Issue list:

--- 
This is a  📊Tracking Issue

**Comment by rambej0201** [comment]
Hi everybody, on one of my data pipeline I am trying to implement time base partition for s3-target-endpoint as like we have TimeBasedPartitioner in io.confluent.connect.s3.S3SinkConnector

Do we have any related implementations or issues going on.

Thanks
Raghu

**Comment by misterjoshua** [comment]
I'd use this. It would be nice if I could create both the migration task and replication instance in one go. In the case of a one-time migration, it would be nice if the construct deleted both the migration task and the replication instance after it is done.

**Comment by jogold** [comment]
Anyone working on this?

**Comment by misterjoshua** [comment]
> Anyone working on this?

Not really. I have some TS types for the migration schema format kicking around somewhere if you want to take a look.
